code/game/gamemodes/cult/ritual.dm

```diff
@@ -178,11 +124,8 @@ It also contains rune words, which are soon to be removed.
 	rune will draw a small amount of life force from the user and make the space above the rune completely dense, rendering it impassable to all but the most complex means. The rune may be invoked again to \
 	undo this effect and allow passage again.<br><br>"
 
-	text += "<font color='red'><b>Deafen</b></font><br>The Rite of the Unheard Whisper is simple. When invoked, it will cause all non-cultists within a radius of seven tiles to become \
-	completely deaf for a large amount of time.<br><br>"
-
-	text += "<font color='red'><b>Blind</b></font><br>Much like the Rite of the Unheard Whisper, the Rite of the Unseen Glance serves a single purpose. Any non-cultists who can see \
-	the rune will instantly be blinded for a substantial amount of time.<br><br>"
+	text += "<font color='red'><b>Debilitate</b></font><br>The Rite of the Shadowed Mind is simple. When invoked, it will cause all non-cultists that can see its rune to become deaf, blind and mute for a \
+	considerable amount of time.<br><br>"
 
 	text += "<font color='red'><b>Stun</b></font><br>Though the Rite of Blazing Light is weak when invoked normally, using it in conjuction with the Rite of Binding makes it much more useful. \

```

Comment:
You can't make stunpapers anymore so the "use this with imbue to stun the crew forever" is wrong. You should note that it stuns on cross now instead, something like this;

    Though the Rite of Blazing Light is weak when invoked normally, merely disorienting and disabling nearby non-cultists, non-cultists that step on the rune will be stunned without expending its power.
    However, the disorientation caused by this rune is quite brief, and when invoked directly, it will not remain for further invocation.